PopPUNK/web.py

```diff
@@ -0,0 +1,230 @@
+import json
+import h5py
+import os 
+import re
+import sys
+import numpy as np
+import pandas as pd
+import graph_tool.topology
```

Comment:
For consistency with other files
```suggestion
import graph_tool.all as gt
```